### DISCUSSION TITLE
Are passwords cropped down to 12?

### DISCUSSION THREAD

**Discussion by Kaleb-13** [jmstoolbox/jmstoolbox#178] [General]
My company issued a new policy, that means that whenever we get issued a fresh password, what we get is 14 positions.
Suddenly JMS could not connect with our queue managers, and after a bit of tinkering, it looks to me that JMS crops down the paswqord to 12 and sends *that* to my MQ manager (when my password is 12 positions, I can add extra characters and it still works).

Not only would a warning that the user entered an PW of more than the max 12 positons be a nice2havefeature, is that max of 12 really unavoidable?

P.S.   MQ explorer works with a PW14, so it's not on the ibmMQ end.

(JMS (5.6.0) / Win10(22h2) / imbmq(9.0.0.9 p900-009-200303)

**Comment by titou10titou10** [comment]
JTB does not have such restrictions, this is not a JTB issue but an MQ issue/configuration setting

https://www.ibm.com/support/pages/mq-when-using-ldap-mqcsp-feature-allows-pass-userids-and-passwords-are-longer-12-characters

https://www.ibm.com/support/pages/when-userid-more-12-characters-invokes-ibm-mq-administration-commands-fdcs-are-generated-xy051169-initprivateservices-xcsgetpwuid-user-name-unknown

...

**Reply by Kaleb-13** [reply]
Thanks !
Especially for the brilliant links.
